lib/vanagon/project.rb

```diff
@@ -162,12 +162,20 @@ def merged_environment
     # Collects all sources and patches into the provided workdir
     #
     # @param workdir [String] directory to stage sources into
-    def fetch_sources(workdir)
+    # @param retry_count [Number] number of times to retry each fetch
+    # @param timeout [Number] How long to wait for each fetch before aborting
```

Comment:
maybe worth adding that time is in seconds?

also, does yardoc like [Number] over [Integer]?

Reply:
Apparently 'Integer'. I honestly just guessed at the base class of number types in ruby (turns out its `Numeric`, which is an adjective not a noun -_- )

Reply:
of course